specification/quantum/Quantum.Workspace/operations/jobs.tsp

```diff
@@ -0,0 +1,152 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+import "./operations.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+
+using TypeSpec.Http;
+using OpenAPI;
+
+/** The type of the job. */
+enum JobType {
```

Comment:
Heads up. extensible enum will be changing to be modelled with union. Can do after this PR is in.

Reply:
Sounds good. I'll do this in the next PR.